### DISCUSSION TITLE
Control more that one Climate

### DISCUSSION THREAD

**Discussion by gvhomeassistant** [Blackymas/NSPanel_HA_Blueprint#1094] [Ideas]
Hi
I would love to be able to control all my smart thermostats from one central point (NSpanel)
I have 3 different thermostats over the place and would be great to be able to adjust all of them from my NSPanel
Thanks !

**Comment by edwardtfn** [comment]
You can assign each thermostat to a different button either in the Home page or any of the button pages and then control those.

**Comment by gvhomeassistant** [comment]
Actually, it seems to work on the home page, but I can't find climate
device in the other pages button setup

Le dim. 24 sept. 2023, 15 h 06, Edward Firmo ***@***.***> a
écrit :

**Reply by edwardtfn** [reply]
Any of the buttons on the buttons pages will accept a climate entity in the "Entity" input.
You have 32 of those. 😉
